### PR TITLE
[CI] Provide the info of the commit that generated a message.

### PR DESCRIPTION
**Pull request by mandel-macaque**
Will add an extra line with the commit message and the hash.

### CODE REVIEW

**Review by dalexsoto — APPROVED** [review]

**Review by spouliot — APPROVED** [review]

File: tools/devops/automation/scripts/GitHub.psm1 [hunk 1]
```diff
@@ -256,6 +256,7 @@ function New-GitHubComment {
     }
     $msg.AppendLine()
     $msg.AppendLine("[Pipeline]($targetUrl) on Agent $Env:TESTS_BOT") # Env:TESTS_BOT is added by the pipeline as a variable coming from the execute tests job
+    $msg.AppendLine("$Env:BUILD_SOURCEVERSIONMESSAGE $Env:BUILD_SOURCEVERSION") # default envars to provide more context to the result
```
Comment (mandel-macaque) [inline tools/devops/automation/scripts/GitHub.psm1:259]:
```suggestion
    $msg.AppendLine("$Env:BUILD_SOURCEVERSIONMESSAGE") # default envars to provide more context to the result
```

**Comment by mandel-macaque** [comment]
Failing tests is related to network issues we have been having in the lab:
```
GHIssue8342(Unauthorized,"mandel","12345678","","") : Status not ok 
```

**Pull request merged**